tests/src/Phansible/Roles/RoleTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Phansible\Roles;
+
+class RoleTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @use Phansible\Roles\BaseRole::getInitialValues
+     */
+    public function testInitialValuesShouldReturnArray()
+    {
+        $role = $this->getMockBuilder('\Phansible\BaseRole')
+            ->disableOriginalConstructor()
+            ->getMockForAbstractClass();
+
+        $role->expects($this->once())
+            ->method('getInitialValues')
+            ->will($this->returnValue([]));
+
+        $this->assertInternalType('array', $role->getInitialValues());
```

Comment:
Sorry... my bad. I was trying to avoid test duplication through abstracting all BaseRole implementations in only one test. But it seems I need to create one test for each implementation.

Thanks.